scripts/checks.m4

```diff
@@ -384,49 +384,6 @@ AC_DEFUN([TORRENT_WITH_FASTCGI], [
     ])
 ])
 
-
-AC_DEFUN([TORRENT_WITH_XMLRPC_C], [
-  AC_MSG_CHECKING(for XMLRPC-C)
-
-  AC_ARG_WITH(xmlrpc-c,
-    AS_HELP_STRING([--with-xmlrpc-c=PATH],[enable XMLRPC-C support]),
-  [
-    if test "$withval" = "no"; then
-      AC_MSG_RESULT(no)
-
-    else
-      if test "$withval" = "yes"; then
-        xmlrpc_cc_prg="xmlrpc-c-config"
-      else
-        xmlrpc_cc_prg="$withval"
-      fi
-      
-      if eval $xmlrpc_cc_prg --version 2>/dev/null >/dev/null; then
-        CXXFLAGS="$CXXFLAGS `$xmlrpc_cc_prg --cflags server-util`"
-        LIBS="$LIBS `$xmlrpc_cc_prg server-util --libs`"
-
-        AC_LINK_IFELSE([AC_LANG_PROGRAM([[ #include <xmlrpc-c/server.h>
-        ]], [[ xmlrpc_registry_new(NULL); ]])],[
-          AC_MSG_RESULT(ok)
-        ],[
-          AC_MSG_RESULT(failed)
-          AC_MSG_ERROR(Could not compile XMLRPC-C test.)
-        ])
-
-        AC_DEFINE(HAVE_XMLRPC_C, 1, Support for XMLRPC-C.)
-
-      else
-        AC_MSG_RESULT(failed)
-        AC_MSG_ERROR(Could not compile XMLRPC-C test.)
-      fi
-    fi
-
-  ],[
-    AC_MSG_RESULT(ignored)
-  ])
-])
-
-
```

Comment:
Could we still keep `xmlrpc-c` as an option? Then add `tinyxml2` as an alternative option? At least ten thousand users are running `ruTorrent` with `xmlrpc-c` currently. We need time to test this before dropping `xmlrpc-c`.

Reply:
Re-added xmlrpc-c support, the exact details of what should be enabled by default is being discussed in the main comments